data/tribes/ships/amazons/init.lua

```diff
@@ -51,6 +51,364 @@ descriptions:new_ship_type {
       pgettext("shipname", "Iscuandé"),
       -- TRANSLATORS: This Amazonian ship is named after a river
       pgettext("shipname", "Putumayo"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Acari"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Acre"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Açuã"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Acurauá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Aiari"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Alalaú"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Amanã"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Amapá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Amazon"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Andirá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Andirá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Antimary"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Apoquitaua"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Apuaú"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Araçá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Arara"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Ararirá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Arauá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Arauá"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
+      pgettext("shipname", "Arauã"),
+      -- TRANSLATORS: This Amazonian ship is named after a river
```

Comment:
To of same name, last differs only with the hyphen on last 'a'. I think during a game no one can distinguish between apostroph and hyphen?